github/github.go

```diff
@@ -856,14 +856,23 @@ func (t *UnauthenticatedRateLimitedTransport) RoundTrip(req *http.Request) (*htt
 	// To set extra querystring params, we must make a copy of the Request so
 	// that we don't modify the Request we were given. This is required by the
 	// specification of http.RoundTripper.
-	req = cloneRequest(req)
-	q := req.URL.Query()
+	//
+	// Since we are going to modify only req.URL here, we only need a deep copy
+	// of req.URL.
+	// shallow copy of request
+	req2 := new(http.Request)
+	*req2 = *req
+	// deep copy of request URL
```

Comment:
The "shallow copy of request" and "deep copy of request URL" phrases seem out of place here, since there's a sentence explaining it on top. Let's get rid of them.